### PR TITLE
Cleaner header and footer for bundle landing pages

### PR DESCRIPTION
**Pull request by JustinPinner**
## Why are you doing this?
We wanted cleaner headers and footers on the landing pages.

## Screenshots
screen shot 2017-01-27 at 15 30 09
.
.
screen shot 2017-01-27 at 15 30 32

@svillafe I think you've been waiting for this.

### CODE REVIEW

File: frontend/app/views/fragments/global/header.scala.html [hunk 1]
```diff
@@ -6,18 +6,44 @@
...
...v-18c0-7-5.2-9.4-13.3-9.4-5.3 0-8.6 1.3-11.4 2.6v7.8h4.7l2-6c1.1-.4 2.3-.4 2.7-.4zm2.3 22.9c-1.9 0-4-1.1-4-4.6 0-2.4 2.4-4.7 4.8-5l2.2-.5v8.5s-1.9 1.6-3 1.6zm100.7-22.9c3.9 0 5.3 2 5.3 5.9v3.5l-6.1 1.1c-5.9 1.1-10.4 2.9-10.4 9.3 0 5.1 3.5 8.7 8.3 8.7 3.8 0 7.4-1.7 8.7-4.4h.3c.5 3.3 3.3 4.4 6.4 4.4 2.4 0 4.8-.6 5.7-1.6v-2l-2.9-1.5v-18c0-7-5.2-9.4-13.3-9.4-5.3 0-8.6 1.3-11.4 2.6v7.8h4.7l2-6c1.1-.4 2.3-.4 2.7-.4zm2.3 22.9c-1.9 0-4-1.1-4-4.6 0-2.4 2.4-4.7 4.8-5l2.2-.5v8.5s-1.9 1.6-3 1.6zm-138.7 5.6c.4 0 .9 0 1.3-.1 3.5-.3 6.7-2 8.4-4.2v4.1l12-1.5v-2l-3-2V13h-1l-12 2.3V18l4 1.7V38c-1.1.8-2.4 1.3-4.2 1.3-2.5 0-4.8-.8-4.8-4.3V13h-1l-12 2.5v2.6l4 1.6V36c0 5.4 2.2 9.7 8.3 9.7zM96 38c-1.2 0-2.5-.8-2.5-1.9 0-.8.6-1.7 1.4-2.3 1.6.5 3 .6 5 .6 7.8 0 13.2-3.7 13.2-10.4 0-3-1.3-4.6-3.2-6.4L115 19v-6l-8.2 1.6c-1.9-.7-4.5-1.6-7-1.6-7.8 0-13.2 4.1-13.2 10.8 0 4.1 2 7.1 5 8.8l.2.2c-1.7 1.2-5.3 4-5.3 7.2 0 2.4 1.5 4.8 4.8 5.5-3.4.8-7.3 2.5-7.3 6.5 0 4.1 5.9 8 15.5 8 11.8 0 16.5-5.7 16.5-13 0-6.1-2.8-9-9.5-9H96zm7.5-14c0 5.7-1.3 6.5-3.5 6.5s-4-.8-4-6.5c0-5.8 1.8-7.5 4-7.5s3.5 2 3.5 7.5zM92 50.9c.1-1.5 1.1-3.4 3.7-3.6h8.6c2.5 0 3.7 2 3.7 3.6 0 3.2-2 4.4-8.3 4.4-5.5 0-7.8-2.2-7.7-4.4z" fill="#fff"></path>
```
Comment (svillafe) [inline frontend/app/views/fragments/global/header.scala.html]: This `svg` should be in a different file

File: frontend/assets/stylesheets/components/_bundle-footer.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+.global-footer-bundles {
+    .global-brandbar {
+        background: none;
+        background-color: #005689;
+    }
+    .global-brandbar__logo {
+        display: none;
+    }
+    .control,
```
Comment (svillafe) [inline frontend/assets/stylesheets/components/_bundle-footer.scss]: The CSS is not using BEM

File: frontend/app/views/fragments/global/header.scala.html [hunk 3]
```diff
@@ -6,18 +6,44 @@
 @import configuration.Config
 @import model.SVG.Logos
 
+@if(!pageInfo.url.startsWith("/bundles")) {
```
Comment (svillafe) [inline frontend/app/views/fragments/global/header.scala.html]: Check the possibility of extending the behaviour  built by @Ap0c (in the PR https://github.com/guardian/membership-frontend/pull/1368), maybe by creating a different cases of the header.

**Comment by JustinPinner** [comment]
Don't get me wrong, this was a quick and dirty hack to get it ready in time ☠️

**Comment by Ap0c** [comment]
Updated based on input from Ben, now looks like this:

bundle-header
...
bundle-footer

Yeah @JustinPinner, makes sense, we were just gna tweak it with these changes and merge. We'll have a look on Monday and see if we can rework it a bit, but if it looks like it'll take too long we can merge it like this, it's perfectly fine.

**Pull request merged**